project-addons/flask_middleware_connector/events/product_events.py

```diff
@@ -143,6 +143,17 @@ def delay_export_product_write(session, model_name, record_id, vals):
         if field in vals:
             update_product.delay(session, model_name, record_id, priority=2, eta=30)
             break
+    is_pack = session.env['product.pack.line'].search([('product_id', '=', record_id)])
+    if is_pack:
+        pack_products = session.env['product.pack.line'].search([('parent_product_id', '=', is_pack.parent_product_id.id)])
+        min_stock = False
+        for product in pack_products:
```

Comment:
el search no es necesario se haría:
for product in is_pack.parent_product_id.pack_line_ids: